src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -308,9 +314,14 @@ public static boolean isValidDirectory(final Path p) {
                     ? configFilePath.toUri().toString()
                     : resolveFilePathStringFromKnownPath( dataSourceProperties.getProperty(CONFIG_FILE_FIELD_NAME_SRC_FILE), configFilePath ).toUri().toString();
 
+        final int lookaheadCacheSizePropertyValue = getLookAheadCacheBpPropertyValue(dataSourceProperties);
+        final int lookaheadCacheSize_bp = lookaheadCacheSizePropertyValue == -1 ? lookaheadFeatureCachingInBp : lookaheadCacheSizePropertyValue;
```

Comment:
Typically, we do not use underscores in Java variable names.   Suggestion: Can you append "Final" or some other indicator that this is the final value you will use?

Reply:
The `_bp` is to indicate the units of the variable.  While camel case is the norm, I think it's more readable to include the units in underscores after the variable name (though I may be inconsistent in the code).

I can change it.